e2e-tests/playwright/e2e/plugins/bulk-import.spec.ts

```diff
@@ -18,7 +18,7 @@ test.describe.serial('Bulk Import plugin', () => {
   let uiHelper: UIhelper;
   let common: Common;
   let bulkimport: BulkImport;
-  let backstageShowcase: BackstageShowcase;
+  let backstageShowcase: BackstageShowcase; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
@albarbaro you may remove this line, as its not being used in this test anymore.